### PR TITLE
AI PL Launch - Create new How AI Works Page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Create a https://code.org/ai/how-ai-works landing page to highlight new How AI Works lessons

**Related PR, depends on:**
- https://github.com/code-dot-org/code-dot-org/pull/53136

**Jira ticket:** [ACQ-757](https://codedotorg.atlassian.net/browse/ACQ-757)

----

how-ai-works-page

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/ai/how-ai-works.haml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    %p=hoc_s(:how_ai_works_lessons_in_a_day_desc)
+    .action-block.action-block--two-col
+      %figure.col-50
+        = view :display_video_thumbnail, id: "introducing_ai", video_code: "Ok-xpKjKp2g", play_button: 'center', letterbox: 'false', download_path: "//videos.code.org/ai_series/ai-01-intro-to-ai.mp4"
+      .text-wrapper.col-50
+        %p=hoc_s(:how_ai_works_lessons_in_a_day_desc_02)
+        %p
+          %strong=hoc_s(:module_label_duration)
+          -# =hoc_s(:module_duration_1_hr)
+          1 hour
```
Comment (megcrenshaw) [inline pegasus/sites.v3/code.org/public/ai/how-ai-works.haml:54]: Does this need to be translated?

File: pegasus/sites.v3/code.org/public/images/banners/banner-bg-maze-dots-light.png
Comment (megcrenshaw) [inline pegasus/sites.v3/code.org/public/images/banners/banner-bg-maze-dots-light.png:1]: I don't see what's being added here, but I'm trusting this is a real image?

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -230,6 +236,15 @@
...
   aria_label_call_to_action_csf_course_f: "See Course F details"
+  aria_label_call_to_action_ai_pl_101_videos: "Watch AI 101 for Teachers videos"
+  aria_label_call_to_action_ai_teaching_module: "Explore the Teaching AI and Machine Learning module"
+  aria_label_call_to_action_how_ai_works_lesson_01: "See the Introduction to Machine Learning lesson details"
+  aria_label_call_to_action_how_ai_works_lesson_02: "See the Computer Vision lesson details"
+  aria_label_call_to_action_how_ai_works_lesson_03: "See the Neural Networks lesson details"
+  aria_label_call_to_action_how_ai_works_lesson_04: "See the Chatbots and Large Language Models lesson details"
+  aria_label_call_to_action_how_ai_works_lesson_05: "See the Generative Images lesson details"
+  aria_label_call_to_action_how_ai_works_lesson_06: "See the Algorithmic Bias lesson details"
+  aria_label_call_to_action_how_ai_works_lesson_07: "See the Our AI Code of Ethics lesson details"
```
Comment (megcrenshaw) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:247]: Thanks for adding these!

**Review by megcrenshaw — APPROVED** [review]
Two small requests, but nothing blocking! Looks great, as always! 🥳

File: pegasus/sites.v3/code.org/public/images/banners/banner-bg-maze-dots-light.png
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/images/banners/banner-bg-maze-dots-light.png:1]: Oh yes, sorry! I forgot to comment that it's the maze pattern image 😅 It's translucent white haha.

File: pegasus/sites.v3/code.org/public/ai/how-ai-works.haml (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/ai/how-ai-works.haml:54]: Yep! This is the thing I was talking about in check-in earlier. Those strings are in [this PR](https://github.com/code-dot-org/code-dot-org/pull/53136), so once that gets merged I can un-comment these and remove the normal text.

Comment (megcrenshaw) [reply pegasus/sites.v3/code.org/public/ai/how-ai-works.haml:54]: Ah, yes! Makes sense 👍

File: pegasus/sites.v3/code.org/public/images/banners/banner-bg-maze-dots-light.png
Comment (megcrenshaw) [reply pegasus/sites.v3/code.org/public/images/banners/banner-bg-maze-dots-light.png:1]: Ah I see it! Had to put it on a different background. Thanks!

**Pull request merged**